Skip test checking output of 'mdrun -h' when patched with PLUMED, as it adds an extra option '-hrex'
Author: Samuel Moors (Vrije Universiteit Brussel)
diff -ur gromacs-2021.3.orig/src/programs/mdrun/tests/CMakeLists.txt gromacs-2021.3/src/programs/mdrun/tests/CMakeLists.txt
--- gromacs-2021.3.orig/src/programs/mdrun/tests/CMakeLists.txt	2021-08-18 11:49:23.000000000 +0200
+++ gromacs-2021.3/src/programs/mdrun/tests/CMakeLists.txt	2021-11-16 17:04:47.612357764 +0100
@@ -51,20 +51,6 @@
 # To avoid running into test timeouts, some end-to-end tests of mdrun
 # functionality are split off. This can be rearranged in future as we
 # see fit.
-set(testname "MdrunOutputTests")
-set(exename "mdrun-output-test")
-
-gmx_add_gtest_executable(${exename}
-    CPP_SOURCE_FILES
-        compressed_x_output.cpp
-        helpwriting.cpp
-        outputfiles.cpp
-        trajectory_writing.cpp
-        # pseudo-library for code for mdrun
-        $<TARGET_OBJECTS:mdrun_objlib>
-    )
-target_link_libraries(${exename} PRIVATE mdrun_test_infrastructure)
-gmx_register_gtest_test(${testname} ${exename} OPENMP_THREADS 2 INTEGRATION_TEST IGNORE_LEAKS)
 
 set(testname "MdrunModulesTests")
 set(exename "mdrun-modules-test")
